Kudu.Contracts/Deployment/IDeploymentManager.cs

```diff
@@ -1,8 +1,8 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
-using Kudu.Core.SourceControl;
 using Kudu.Contracts.Tracing;
+using Kudu.Core.SourceControl;
```

Comment:
Do we need this ?

Reply:
Yes it is used for several class types in the document.